TASVideos/Pages/Shared/Components/TabularMovieList/Default.cshtml

```diff
@@ -24,7 +24,7 @@
 					{
 						<tr>
 							<td><timezone-convert asp-for="@pub.CreateTimestamp" date-only="true" /></td>
-							<td>@pub.Game</td>
+							<td>@pub.Game<span condition="pub.Branch is not null"> "@(pub.Branch)"</span></td>
```

Comment:
probably safer (more paranoid) to use !string.IsNullOrWhiteSpace(pub.Branch)

Reply:
*Ah*, yeah I saw it being used every now and then and never read what it actually means. Will change.